components/Sighting.jsx

```diff
@@ -1,19 +1,101 @@
 import { StatusBar } from "expo-status-bar";
-import { StyleSheet, Text, View } from "react-native";
+import dayjs from "dayjs";
+import {
+  StyleSheet,
+  Text,
+  Image,
+  Dimensions,
+  ScrollView,
+  Box,
+  View,
+} from "react-native";
 
-export default Sighting = ({ navigation }) => {
+let width = Dimensions.get("window").width;
+
+export default Sighting = ({ route, navigation }) => {
+  const {
+    common_name,
+    bird_image_url,
+    date_spotted,
+    screen_name,
+    rarity,
+    coordinates,
+  } = route.params;
+  const dateDay = dayjs(date_spotted).format("DD-MM-YYYY");
+  const dateTime = dayjs(date_spotted).format("HH:mm:ss");
   return (
-    <View styles={styles.container}>
-      <Text>Sighting page can go here...</Text>
+    <View style={styles.container}>
+      <Text style={styles.birdName}>{common_name}</Text>
+      <Image
+        source={{
+          uri: bird_image_url,
+        }}
+        style={styles.image}
+      />
+      <Text style={styles.birdInfo}>
+        Date Spotted: {dateDay} at {dateTime}
+      </Text>
+      <Text style={styles.birdInfo}>
+        Location of Sighting:{" "}
+        {coordinates.coordinates.map((point) => `${point}, `)}
+      </Text>
+      <Text style={styles.birdInfo}>Rarity: {rarity} </Text>
+      <Text style={styles.birdInfo}>Spotted by: {screen_name}</Text>
     </View>
+    //   <ScrollView>
```

Comment:
Could get rid of this comment block if not needed

Reply:
with regards to the data I think this can be refactored in a different phase if necessary but it does sound like a good idea to me!

Updated under "
Update-to-Sighting"

Cheers Cam!

Reply:
Okay sounds good. I'll create a ticket in the Kanban board to refactor later.
Otherwise g2g!